static/app/views/explore/tables/tracesTable/fieldRenderers.tsx

```diff
@@ -57,72 +57,58 @@ export function SpanDescriptionRenderer({span}: {span: SpanResult<Field>}) {
 interface ProjectsRendererProps {
   projectSlugs: string[];
   maxVisibleProjects?: number;
+  visibleAvatarSize?: number;
 }
 
 export function ProjectsRenderer({
   projectSlugs,
+  visibleAvatarSize,
   maxVisibleProjects = 2,
 }: ProjectsRendererProps) {
   const organization = useOrganization();
+  const {projects} = useProjects({slugs: projectSlugs, orgId: organization.slug});
```

Comment:
Moving away from using the deprecated `<Projects>{children}</Projects>` wrapper and using `useProjects` instead. It provides an incorrect list of projects when you navigate back to a page.